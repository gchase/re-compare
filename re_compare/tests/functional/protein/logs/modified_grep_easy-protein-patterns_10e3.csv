regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,3530,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,3530,"[-1, -1]"
DHKNLD.,first_match,0,607,"[-1, -1]"
DHKNLD.,all_matches,0,607,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,0,1482,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,0,1482,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,690,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,690,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,929,"[2, 2]"
G|S|T|A|D|N|E|K|R,consecutive,1,929,"[2, 2]"
G|S|T|A|D|N|E|K|R,consecutive,2,4,"[3, 3]"
G|S|T|A|D|N|E|K|R,all_matches,2,933,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,0,775,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,0,775,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,686,"[121, 122]"
(N|S.TNH.Y)(F|W),consecutive,1,686,"[121, 122]"
(N|S.TNH.Y)(F|W),consecutive,2,7,"[440, 441]"
(N|S.TNH.Y)(F|W),all_matches,2,693,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,0,673,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,0,673,"[-1, -1]"
P..L...KWE.,first_match,0,651,"[-1, -1]"
P..L...KWE.,all_matches,0,651,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,635,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,635,"[-1, -1]"
