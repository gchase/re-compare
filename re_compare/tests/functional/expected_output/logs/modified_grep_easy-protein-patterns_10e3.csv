regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,2821,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,2821,"[-1, -1]"
DHKNLD.,first_match,0,671,"[-1, -1]"
DHKNLD.,all_matches,0,671,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,0,1165,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,0,1165,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,663,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,663,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,2090,"[2, 2]"
G|S|T|A|D|N|E|K|R,consecutive,1,2090,"[2, 2]"
G|S|T|A|D|N|E|K|R,consecutive,2,71,"[3, 3]"
G|S|T|A|D|N|E|K|R,all_matches,2,2161,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,0,819,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,0,819,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,868,"[121, 122]"
(N|S.TNH.Y)(F|W),consecutive,1,868,"[121, 122]"
(N|S.TNH.Y)(F|W),consecutive,2,8,"[440, 441]"
(N|S.TNH.Y)(F|W),all_matches,2,876,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,0,665,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,0,665,"[-1, -1]"
P..L...KWE.,first_match,0,630,"[-1, -1]"
P..L...KWE.,all_matches,0,630,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,629,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,629,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),first_match,1,1144,"[829, 837]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,1,1144,"[829, 837]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),all_matches,1,1161,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),first_match,0,936,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),all_matches,0,936,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),first_match,0,775,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),all_matches,0,775,"[-1, -1]"
A|G....G,first_match,1,983,"[14, 14]"
A|G....G,consecutive,1,983,"[14, 14]"
A|G....G,consecutive,2,8,"[18, 18]"
A|G....G,all_matches,2,991,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,first_match,0,666,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,all_matches,0,666,"[-1, -1]"
